scripts/prodsec-manifest-generator.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+# This script generates a manifest of the integreatly-operator which includes the direct and indirect dependencies 
+# used. The generated manifest corresponds to a released version and is located in : ../rhoam-manifests/
+
+# Compares current manifest vs manifest generated for master
+manifest_compare() {
+    # Additional sorting of the master manifest is required as it seems that prow sorts files in a different manner.
+    sort -u "$RHOAM_CURRENT_MASTER" > "$RHOAM_SORTED_MASTER"
+
+    MANIFESTS_DIFF=$(diff --suppress-common-lines ${RHOAM_MASTER_FROM_BRANCH} ${RHOAM_SORTED_MASTER})
+    if [ ! -z "$MANIFESTS_DIFF" ]; then
+        echo "Difference found between master manifests, run `TYPE_OF_MANIFEST=master make manifest/prodsec` and push PR again"
```

Comment:
Can we remove the `"and push PR again"`, This command can be call in cases out side of a PR

Reply:
As agreed, resolving this conversation.